packages/website/src/content/docs/setup/other.mdx

```diff
@@ -104,7 +106,7 @@ Sentry.init({
 
 ---
 
-## Run the Sidecar
+## 3. Run the Sidecar
```

Comment:
Just noticed (can't mark the line though in the GH UI): Can we remove the `:::tip` block in lines 91-103? I think this is essentially a duplicate of the newer "Leverage Sentry SDKs" section

Reply:
Done, removed. the Leverage Sentry SDK section is fine. no need of this tip.